pages/index.tsx

```diff
@@ -38,6 +38,31 @@ import { useTranslation } from "next-i18next"
 import AboutModal from "../components/AboutModal"
 import Layout from "../components/Layout"
 import { FaMoon, FaSun } from "react-icons/fa"
+import { IconType } from "react-icons"
+import { theme } from "./_app"
+
+/**
+ * Darkens a color by a percentage
```

Comment:
This looks good for now, but we can probably change the Box into a Button in the near future.